.github/workflows/MacOS.yml

```diff
@@ -0,0 +1,53 @@
+name: MacOS
+on:
+  push:
+    branches:
+    - develop
+    paths-ignore:
+    - README.md
+  pull_request:
+    branches:
+    - develop
+    paths-ignore:
+    - README.md
+
+jobs:
+  MacOS:
+    runs-on: macos-latest
+    env:
+      FC: gfortran-11
+
+    steps:
+    
+    - name: install-deps
+      run: |
+        brew update
+        brew install doxygen
+        pip3 install setuptools
+        pip3 install numpy
+        pip3 install netCDF4
+        sudo ln -sf /usr/local/bin/gfortran-11 /usr/local/bin/gfortran
+          
+    - name: checkout-bufr
+      uses: actions/checkout@v2
+      with: 
+        path: bufr
+
+    - name: build-bufr
+      run: |
+        cd bufr
+        mkdir build 
+        cd build
+        cmake ..
+        make -j2 VERBOSE=1
+        cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_PYTHON=ON ..
+        make -j2 VERBOSE=1
+        make install
+        cd python
+        sudo python3 setup.py install
+
+    - name: test-bufr
+      run: |
+        cd bufr/build
```

Comment:
I noticed in some of the yml files this was just `cd bufr/build` and in others it was `cd $GITHUB_WORKSPACE/bufr/build`.  Which one is correct?

Even if they both work, I think consistency among all of them would be helpful ;-)

Reply:
Fixed.